VSQlik: Show Connection Settings is blank

**Describe the bug**
Running the command `VSQlik: Show Connection Settings` results in a blank page.

**To Reproduce**
1. Show commands (Shift-Ctrl-P)
2. Select `VSQlik: Show Connection Settings`
3. The settings page is blank

**Expected behavior**
The settings page would not be blank.

**Screenshots**
![Blank Settings Page](https://user-images.githubusercontent.com/55477181/85494309-bf6a2c00-b61b-11ea-95b6-1f953c4ae43a.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro for Workstations, version 1909
 - VS Code version: 1.46.1


**Additional context**
The Console in 'Help -> Toggle Developer Tools' says:

```
webviewElement.ts:51 Webview: Cannot load resource outside of protocol root
(anonymous) @ webviewElement.ts:51
async function (async)
(anonymous) @ webviewElement.ts:43
apply @ callbacks-registry.ts:45
(anonymous) @ remote.js:286
(anonymous) @ remote.js:276
emit @ events.js:203
onMessage @ init.ts:59
```

@r-hannuschka still happening in version 0.3.0

Error: Unable to read file '\.\styles.0220be4e5426a6fb155d.css' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open 'C:\styles.0220be4e5426a6fb155d.css')
    at e.doReadAsFileStream (fileService.ts:457)
resourceLoader.ts:59 Error: Unable to read file '\.\runtime-es2015.0dae8cbc97194c7caed4.js' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open 'C:\runtime-es2015.0dae8cbc97194c7caed4.js')
    at e.doReadAsFileStream (fileService.ts:457)
resourceLoader.ts:59 Error: Unable to read file '\.\polyfills-es2015.9ad3fb84653bac45550c.js' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open 'C:\polyfills-es2015.9ad3fb84653bac45550c.js')
    at e.doReadAsFileStream (fileService.ts:457)
resourceLoader.ts:59 Error: Unable to read file '\.\main-es2015.6b2d4133771fe80dbff8.js' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open 'C:\main-es2015.6b2d4133771fe80dbff8.js')
    at e.doReadAsFileStream (fileService.ts:457)

--- a/src/bin/webapp-finalize.js
+++ b/src/bin/webapp-finalize.js
@@ -1,0 +1,49 @@
+const path = require("path");
+const fs   = require("fs");
+
+const webappPath = path.resolve(process.cwd(), 'dist/webview/connection');
+
+let indexHtml  = fs.readFileSync(path.resolve(webappPath, 'index.html')).toString();
+
+function makeInline(config, data) {
+    let currentData = new Buffer.from('', 'utf-8');
+    let isFlagSet = false;
+
+    return data.replace(config.pattern, (match, script) => {
+        const replacement = !isFlagSet ? '###INLINE_CONTENT###' : '';
+        isFlagSet = true;
+
+        const content = fs.readFileSync(path.resolve(webappPath, script));
+        fs.unlinkSync(path.resolve(webappPath, script));
+
+        if (config.isScript && script.indexOf('-es2015') === -1) {
+            return '';
+        }
+
+        currentData = Buffer.concat([currentData, content]);
+        return replacement;
+    }).replace(/###INLINE_CONTENT###/, () =>  {
+        return config.template.replace(/###INLINE_CONTENT###/, currentData.toString());
+    });
+}
+
+[
+    {
+        isScript: true,
+        pattern: /<script src="(.*?)".*?><\/script>/g,
+        template: `<script type="module">
+        ###INLINE_CONTENT###
+    </script>`
+    },
+    {
+        pattern: /<link rel="stylesheet" href="(.*?)".*?>/g,
+        template: `
+        <style type="text/css">
+        ###INLINE_CONTENT###
+    </style>`
+    }
+].forEach((config) => {
+    indexHtml = makeInline(config, indexHtml);
+});
+
+fs.writeFileSync(path.resolve(webappPath, 'index.html'), indexHtml, {flag: "w"});

--- a/src/package.json
+++ b/src/package.json
@@ -15,6 +15,7 @@
     "postinstall": "npm i --prefix ./projects/media/webview",
     "postbuild": "npm run webview:build",
     "webview:build": "npm run build --prefix ./projects/media/webview -- --prod",
+    "postwebview:build": "node ./bin/webapp-finalize.js",
     "webview:dev": "npm start --prefix ./projects/media/webview -- --configuration browser",
     "changelog:compile": "tsc -p ./ci",
     "changelog:generate": "node ./dist/ci/changelog",

--- a/src/projects/media/webview/projects/connection/src/app/ui/edit/edit.html
+++ b/src/projects/media/webview/projects/connection/src/app/ui/edit/edit.html
@@ -1,5 +1,9 @@
-<div [formGroup]="connectionForm" class="form">
+<div class="btn-bar">
+    <button class="btn" type="button" (click)="doSave()">save</button>
+    <button class="btn" type="button" (click)="doCancel()">cancel</button>
+</div>
 
+<div [formGroup]="connectionForm" class="form">
     <!-- label -->
     <h3>Default Settings</h3>
 
@@ -97,10 +101,4 @@
         <vsqlik-settings--form-strategy *ngSwitchDefault>
         </vsqlik-settings--form-strategy>
     </div>
-
-</div>
-
-<div class="footer btn-bar">
-    <button class="btn" type="button" (click)="doSave()">save</button>
-    <button class="btn" type="button" (click)="doCancel()">cancel</button>
 </div>

--- a/src/projects/media/webview/projects/connection/src/app/ui/main/main.component.html
+++ b/src/projects/media/webview/projects/connection/src/app/ui/main/main.component.html
@@ -8,8 +8,7 @@
         </div>
         <div class="actions">
             <button class="btn action" (click)="createSetting()">
-                <i class="vsqlik-icon vsqlik-icon--add"></i>
-                <span class="label">Add Connection</span>
+                <span class="label">Create Connection</span>
             </button>
         </div>
     </header>

--- a/src/projects/media/webview/projects/connection/src/app/ui/main/main.component.scss
+++ b/src/projects/media/webview/projects/connection/src/app/ui/main/main.component.scss
@@ -39,6 +39,7 @@
             overflow: hidden;
             text-overflow: ellipsis;
             padding: .2rem .5rem;
+            align-items: center;
         }
 
         td {
@@ -47,11 +48,17 @@
 
         .action-col {
             display: flex;
-            flex: 0 0 5rem;
+            flex: 0 0 120px;
             overflow: hidden;
+            justify-content: flex-end;
 
             .btn {
                 margin: 0 .25rem 0 0;
+                padding: .5rem;
+            }
+
+            .btn:last-child {
+              margin: 0;
             }
         }
     }
@@ -63,7 +70,7 @@
     }
 
     vsqlik-wfs--table-row-edit ::ng-deep {
-        
+
         td:not(.action-col) {
             padding: 0;
         }

--- a/src/projects/media/webview/projects/connection/src/app/ui/table/table-row.html
+++ b/src/projects/media/webview/projects/connection/src/app/ui/table/table-row.html
@@ -5,7 +5,7 @@
     <td>{{setting.connection?.secure}}</td>
     <td>{{setting.connection?.allowUntrusted}}</td>
     <td class="action-col">
-        <button type="button" class="btn btn-action" (click)="doEdit()"><i class="vsqlik-icon vsqlik-icon--edit"></i></button>
-        <button type="button" class="btn btn-action" (click)="doDelete()"><i class="vsqlik-icon vsqlik-icon--delete"></i></button>
+        <button type="button" class="btn btn-action" (click)="doEdit()">edit</button>
+        <button type="button" class="btn btn-action" (click)="doDelete()">remove</button>
     </td>
 </tr>

